src/js/tabs/receiver.js

```diff
@@ -16,6 +18,59 @@ TABS.receiver.initialize = function (callback) {
         GUI.active_tab = 'receiver';
     }
 
+    function lookup_passphrase(uidString) {
+        let passphrase = 0;
+        ConfigStorage.get('passphrase_map', function (result) {
+            if (result.passphrase_map) {
+                const passphraseMap = result.passphrase_map;
+                passphrase = passphraseMap[uidString];
+            }
+        });
+        return passphrase;
```

Comment:
There is not reason to use `callback` here, `get` is synchronous and just returns results (it might be there just for backwards compatability I think. Something like this should suffice.
```suggestion
        let { passphrase_map } = ConfigStorage.get('passphrase_map');
        return passphraseMap[uidString] ?? 0;
```